lib/cinch/plugins/karma.rb

```diff
@@ -29,10 +29,12 @@ def listen(m)
         channel = m.channel.name
 
         # Scan messages for multiple karma items
-        m.message.scan(/(\s|\A)(\w+|\(.+?\))(\+\+|--)(\s|\z)/).each do |k|
+        m.message.scan(/(^|\s|\A|\b)(\w+|\(.+?\))(\+\+|--)(\s|\z|$)/).each do |k|
           process_karma(channel, k[1].gsub(/\(|\)/, '').downcase, k[2])
+	  item = k[1].gsub(/\(|\)/, '')
+	  @storage.synced_save(@bot)
+	  m.reply "#{item} now has #{@storage.data[channel][item.downcase]} point(s) of karma"
```

Comment:
please make auto reply with current karma a plugin option, otherwise it's going to be too spammy for some people's channels (for example, mine 😉)